drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/table/pages/FactPatternConstraintsPageTest.java

```diff
@@ -57,6 +58,7 @@
 import static org.mockito.Mockito.when;
 
 @RunWith(GwtMockitoTestRunner.class)
+@Ignore
```

Comment:
Why do we need to ignore this test now!?

Reply:
We shouldn't. I need to check this.

Reply:
What ever broke it and caused me to ignore turned out to be fixed already.